app/src/main/index.js

```diff
@@ -739,7 +744,7 @@ async function main() {
     fs.writeFileSync(appVersionPath, pkg.version)
   }
 
-  checkGaiaCompatibility(gaiacliVersionPath)
+  await checkGaiaCompatibility(gaiacliVersionPath)
```

Comment:
Thanks, I missed that.

Reply:
Finding on that: if you call async functions without await, errors are not passed to the wrapping function. ergo they never surface. :/

Reply:
Yes, I know.  :-(